CONTRIBUTING.md

```diff
@@ -7,17 +7,12 @@
         $ git checkout -b <branch>
 
 3.  Add new functions as separate source files in src/. If the function is
```

Comment:
We have "Include unit tests if adding a function or fixing a bug." in the next list item, but I think it'd be better here. I'll move it.